doc/developers/logging_cpp.rst

```diff
@@ -99,7 +99,7 @@ If a handler makes no writes to the KV, it may be installed as read-only:
     :end-before: SNIPPET_END: install_get
     :dedent: 6
 
-This offers some additional type safety (accidental `put`s or `remove`s will be caught at compile-time) and also enables performance scaling since read-only operations can be executed on any receiving node, whereas writes must always be executed on the primary node.
+This offers some additional type safety (accidental `put`\s or `remove`\s will be caught at compile-time) and also enables performance scaling since read-only operations can be executed on any receiving node, whereas writes must always be executed on the primary node.
```

Comment:
Does something like :cpp:func:`kv::TxView::put()` work here?

Reply:
No, this also gives a Sphinx error that I saw [here](https://github.com/microsoft/CCF/pull/1325/files#diff-27c722e5d73616262ec810f6ccb8ca8d) (and worked around in a different way).